bundles/org.openhab.binding.roku/README.md

```diff
@@ -47,6 +49,8 @@ The following channels are available:
 | programTitle       | String               | The name of the current TV program (ReadOnly).                                                                                                                  |
 | programDescription | String               | The description of the current TV program (ReadOnly).                                                                                                           |
 | programRating      | String               | The TV parental guideline rating of the current TV program (ReadOnly).                                                                                          |
+| power              | Switch               | Controls the power for the TV.                                                                                                                                  |
+| powerState         | String               | The current power state for the TV.                                                                                                                             |
```

Comment:
Can you provide some context here. What are the possible string values? I assume there are strings representing other values than on/off, since otherwise this channel would be fully redundant?

Reply:
powerState provides more context when the device is in an idle state on some devices.  For example, my TV lists as "Ready" when in low power mode.  I've seen references to other states as well across the googleverse.  I included it for those who want specifics. I do agree the power channel will work for most, I'm just the type of person who will provide more info if I have it on something because I have no idea if it will help someone down the line.

Reply:
It might be good to include the possible values in parentheses:
```suggestion
| powerState         | String               | The current power state for the TV (ie PowerOn, DisplayOff, Ready, etc.).                                                                                                               |
```

Reply:
Also "(ReadOnly)" is missing.